### PR TITLE
Replaced delete_contents script with delete_contents_on_destroy

### PR DESCRIPTION
**Pull request by kiran002**
Implementing issue #9

### CODE REVIEW

File: modules/bigquery/variables.tf [hunk 1]
```diff
@@ -30,3 +30,8 @@ variable "location" {
   description = "The location of the storage bucket."
   default     = "US"
 }
+
+variable "delete_contents_on_destroy" {
+  description = "(Optional) If set to true, delete all the tables in the dataset when destroying the resource; otherwise, destroying the resource will fail if tables are present."
+  default     = "false"
```
Comment (morgante) [inline modules/bigquery/variables.tf]: This should default to `true` for parity with the existing implementation.

File: examples/bigquery/billing_account/main.tf [hunk 2]
```diff
@@ -28,8 +28,9 @@ module "log_export" {
...
 module "destination" {
-  source                   = "../../..//modules/bigquery"
-  project_id               = "${var.project_id}"
-  dataset_name             = "bigquery_example"
-  log_sink_writer_identity = "${module.log_export.writer_identity}"
+  source                     = "../../..//modules/bigquery"
+  project_id                 = "${var.project_id}"
+  dataset_name               = "bigquery_example"
+  log_sink_writer_identity   = "${module.log_export.writer_identity}"
+  delete_contents_on_destroy = "true"
```
Comment (morgante) [inline examples/bigquery/billing_account/main.tf]: We can leave it at a `true` default.

**Comment by aaron-lane** [comment]
Fixes #9.

File: examples/bigquery/billing_account/main.tf (hunk 2 above)
Comment (kiran002) [reply examples/bigquery/billing_account/main.tf]: Is it really good idea to delete the contents as a default behavior?

Comment (morgante) [reply examples/bigquery/billing_account/main.tf]: I think we could discuss changing the default later, but it'd be good to release this without making it backwards-incompatible.

Comment (kiran002) [reply examples/bigquery/billing_account/main.tf]: Sure makes sense, let me change it for now

**Pull request merged**